src/Compilers/CSharp/Portable/Syntax/CSharpSyntaxTree.ParsedSyntaxTree.cs

```diff
@@ -33,6 +47,9 @@ internal ParsedSyntaxTree(SourceText textOpt, Encoding encodingOpt, SourceHashAl
                 _path = path ?? string.Empty;
                 _root = cloneRoot ? this.CloneNodeAsRoot(root) : root;
                 _hasCompilationUnitRoot = root.Kind() == SyntaxKind.CompilationUnit;
+
+                _diagnosticOptions = diagnosticOptions ?? EmptyCaseInsensitiveDictionary;
```

Comment:
> _diagnosticOp [](start = 15, length = 14)

Nit: extra blank lines around this initialization only

Reply:
How does this work? This is a normal struct, not a nullable one, hence how is ?? evaluating here? 

---
In reply to: [182268643](https://github.com/dotnet/roslyn/pull/25964#discussion_r182268643) [](ancestors = 182268643)